src/ucp/rndv/rndv_ppln.c

```diff
@@ -239,7 +253,8 @@ static ucs_status_t ucp_proto_rndv_ppln_progress(uct_pending_req_t *uct_req)
         ucp_proto_request_select_proto(freq, &rpriv->frag_proto,
                                        freq->send.state.dt_iter.length);
 
-        ucp_trace_req(req, "send fragment request %p", freq);
+        ucp_trace_req(req, "send fragment req %p offset %zu size %zu", freq,
```

Comment:
req is already printed by ucp_trace_req

Reply:
`req` and `freq` are different requests, IMO it's useful to see their relationship in debug traces

Reply:
ah, so need to print "freq" instead of "req" here